### ISSUE TITLE
[RTL] DropdownMenu (Column Header)

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8826]
### Description

For RTL:
* The arrow for opening the dropdown menu should be on the left
* Dropdown menu should open from right to left
* Sub-menu should open on the left.
* The content of menus should be aligned to the right.
* The submenu's arrows should be on the left.

### How to reproduce:

1. Use this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/bfe85126b63191ca5f6247fc4df6fdbd46683cb4/vanilla/configurations.js#L765-L770):
```js
{
    data: Handsontable.helper.createSpreadsheetData(5, 20),
    colHeaders: true,
    dropdownMenu: true,
    licenseKey: 'non-commercial-and-evaluation'
};
```
2. On column header `B` click the arrow
3. Result:
![image](https://user-images.githubusercontent.com/2893099/135905804-59813288-b0c7-4d1c-974a-57652b0e6a3c.png)

**Comment by aninde** [comment]
@budnix it looks like the current `RTL` branch has not addressed the changes from point 1. in moving the drop-down arrows to the left side. 

I do not see any information on this on GitHub, was it specifically omitted?

**Comment by aninde** [comment]
> I've switched to the `rtl` branch, built and it seems working fine 🤔 

I meant `The arrow for opening the dropdown menu should be on the left` as I understand this button, which opens the entire menu. The entire menu is reversed except for this button.

![Screenshot 2022-02-02 at 16 26 13](https://user-images.githubusercontent.com/30186818/152183969-f622a75a-f344-4d84-b92c-53cdfd89bf77.png)

**Comment by budnix** [comment]
> I meant `The arrow for opening the dropdown menu should be on the left` as I understand this button, which opens the entire menu. The entire menu is reversed except for this button.

Thanks for spotting that. I created a separate issue https://github.com/handsontable/handsontable/issues/9186 as well the fix will be prepared in separate PR.

**Comment by aninde** [comment]
Fixed in the v12.0.0 https://jsfiddle.net/t69meyL0/2/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
